guides/doc-Administering_Red_Hat_Satellite/topics/security-content-automation-protocol.adoc

```diff
@@ -1,9 +1,9 @@
-[id='security-content-automation-protocol_{context}']
+[id="security-content-automation-protocol_{context}"]
```

Comment:
Out of my curiousity - does this change anything or is it just purely cosmetic?

Reply:
IMHO purely cosmetic/ for convenience.